src/eval/evaluate.py

```diff
@@ -68,133 +73,131 @@ def parse_args() -> argparse.Namespace:
     return args
 
 
-def get_datamodule(dataset_path: Path | str, category: str) -> MVTecLOCO:
-    """Get the MVTec LOCO datamodule.
+def get_dataloaders(dataset_path: Path | str, category: str) -> tuple[DataLoader, DataLoader]:
+    """Get the MVTec LOCO dataloader.
 
     Args:
         dataset_path (Path | str): Path to the dataset.
         category (str): Category of the MVTec dataset.
 
     Returns:
-        MVTec: MVTec datamodule.
+        tuple[DataLoader, DataLoader]: Tuple of train and test dataloaders.
     """
     # Create the dataset
-    datamodule = MVTecLOCO(
+    # NOTE: We fix the image size to (256, 256) for consistent evaluation across all models.
+    train_dataset = MVTecLOCODataset(
         root=dataset_path,
         category=category,
-        eval_batch_size=1,
+        split=Split.TRAIN,
+        augmentations=Resize((256, 256)),
+    )
+    test_dataset = MVTecLOCODataset(
+        root=dataset_path,
+        category=category,
+        split=Split.TEST,
+        augmentations=Resize((256, 256)),
+    )
+    train_dataloader = DataLoader(
+        train_dataset, batch_size=1, shuffle=False, num_workers=4, collate_fn=ImageBatch.collate
```

Comment:
As discussed, we will have to use some sort of adaptive batch size to speed up inference. We could also make it configurable by the participants. Then if their evaluation fails due to OOM, they can reduce the batch size. Of course, this is out of scope of this PR :)